flow360/component/simulation/operating_condition/operating_condition.py

```diff
@@ -252,3 +252,31 @@ def mach(self) -> pd.PositiveFloat:
 # pylint: disable=fixme
 # TODO: AutomotiveCondition
 OperatingConditionTypes = Union[GenericReferenceCondition, AerospaceCondition]
+
+
+def operating_condition_from_mach_reynolds(
+    mach: pd.NonNegativeFloat = None,
+    reynolds: pd.PositiveFloat = None,
+    temperature: pd.PositiveFloat = None,
+    alpha: Optional[AngleType] = 0 * u.deg,
+    beta: Optional[AngleType] = 0 * u.deg,
+    reference_mach: Optional[pd.PositiveFloat] = None
+) -> AerospaceCondition:
+    thermal_state = ThermalState(
+        temperature=temperature,
+        material=Air(
+            dynamic_viscosity=Sutherland(
+                reference_temperature=temperature,
+                reference_viscosity=(mach/reynolds),
```

Comment:
what if user will provide reynolds=0, it will crash here. Remember that types provided to function signature will not be validated in normal python. So we need some check